TOPICS.txt

```diff
@@ -63,11 +63,18 @@ Parsing
 Pattern recognition
 Performance
 Randomness
-Reactive programming
 Refactoring
 Regular Expressions
 Searching
 Security
 Sorting
 Text formatting
 Transforming
+
+Programming Style
+---
+Behavior driven development
```

Comment:
According to [Wikipedia](https://en.wikipedia.org/wiki/Behavior-driven_development),  it is written with a dash: Behavior-driven development